packages/transition-backend/src/services/executableJob/__tests__/ExecutableJob.test.ts

```diff
@@ -164,6 +165,7 @@ describe('Test resume running and pending', () => {
         }];
         mockedJobCollection.mockResolvedValueOnce({ jobs: jobsToRun, totalCount: jobsToRun.length })
         expect(await ExecutableJob.enqueueRunningAndPendingJobs(progressEmitter)).toEqual(true);
+        await TestUtils.flushPromises();
```

Comment:
Probably need a comment about why it's added

Reply:
done